pylint_plugins/conf_plugin.py

```diff
@@ -0,0 +1,90 @@
+
+import astroid
+
+from astroid import MANAGER
+from pylint import lint
+
+def __init__(self):
+  pass
+
+
+#didn't change/work on this yet
+def wrap_add_message(func):
+  def add_message(msg_descr, line=None, node=None, args=None, confidence=None):
+    kwargs = {'confidence': confidence} if confidence else {}
+    if (msg_descr != 'no-member' or
+        args[0:2] != ('Module', 'openhtf.conf')):
+      return func(msg_descr, line, node, args, **kwargs)
+    # We're sure it's a no-member warning for the openhtf.conf module.
+    try:
+      if args[2] in node.root()._declared_conf_keys:
+        # We found a declaration for the symbol, just skip this message.
+        return
+    except AttributeError:
+      pass
+
+    # TODO(maddychan): Figure out how to add a custom failure message here.
+    return func('no-member', line, node, args, **kwargs)
+
+  return add_message
+
+#storing a reference to the openhtf.conf module node
+clss = []
```

Comment:
this is one of those rare cases where it's probably better to use `global` than to use lists, so initialize these to `None` and in your `trans_*` methods do `global clss`, etc. so you can assign to it directly.